src/cloudant/client.py

```diff
@@ -55,11 +55,11 @@ class CouchDB(dict):
 
     def __init__(self, user, auth_token, admin_party=False, **kwargs):
         super(CouchDB, self).__init__()
-        self._cloudant_user = user
-        self._cloudant_token = auth_token
-        self._cloudant_session = None
-        self.cloudant_url = kwargs.get('url')
-        self._cloudant_user_header = None
+        self._user = user
+        self._auth_token = auth_token
+        self._client_session = None
+        self.client_url = kwargs.get('url')
```

Comment:
I considered just naming it `url` but stopped short of doing that because I felt like then I would need to rename `database_url` and `document_url` to `url` in the database and document classes.

Giving it some more thought I renamed `client_url` to `server_url` as suggested, reasoning that while the attribute is public in the client it is not annotated as a property while the `database_url` and the `document_url` are.  So they are somewhat different and `server_url` makes more sense than `client_url`.  Done in 901e7d6.